docs/operator-manual/user-management/onelogin.md

```diff
@@ -115,7 +115,7 @@ data:
   url: https://<argocd.myproject.com>
   oidc.config: |
     name: OneLogin
-    issuer: https://openid-connect.onelogin.com/oidc
+    issuer: https://<subdomain>.onelogin.com/oidc/2
```

Comment:
See: https://developers.onelogin.com/openid-connect/api/upgrade for the reasoning behind this and why the April 20th date would have broken anyone attempting to set this up after this date.